src/plugins/paginate/paginate.js

```diff
@@ -249,7 +262,12 @@ $document.on( "click", "." + pagerClass + " a", function()  {
 
 		updateItems( elm );
 		goToPage( elm );
```

Comment:
The function `goToPage()` need to be updated.

When I am using the pagination button, that function is re-adding the aria-current on the list item when it need to be added on the button instead. See line 190 and line 194 to 196.

Reply:
Good catch. @BrahimMahadi add the following:
```suggestion
		goToPage( elm );
		
		$( elm ).trigger( "setfocus.wb" );
		
```